src/_igraph/graphobject.c

```diff
@@ -13373,8 +13322,11 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "  have an out-type and an in-type. In this case, I{types1} contains the\n"
    "  out-types and this parameter contains the in-types in a list or the\n"
    "  name of a vertex attribute. If C{None}, it is assumed to be equal\n"
-   "  to I{types1}.\n\n"
+   "  to I{types1}.\n"
```

Comment:
The second `\n` was deliberate; I wanted to add an extra separator line between the function description and the list of parameters.

Reply:
... but isn't that between one parameter and the next?

Reply:
Ah yes, you are right, sorry.